site_root/input/ImplementationGuide-KidsFirst.json

```diff
@@ -55,15 +55,7 @@
     ],
     "resource": [
       {
-        "description": "Kids First StructureDefinition participant, Base: Patient",
-        "exampleBoolean": false,
-        "name": "Kids First StructureDefinition/participant",
-        "reference": {
-          "reference": "StructureDefinition/participant"
-        }
-      },
-      {
-        "description": "Kids First Patient pt-001, Profiles: participant",
+        "description": "Kids First Patient pt-001, Profiles: kfdrc-patient",
```

Comment:
no phi

Reply:
@fiendish, could you elaboration the comment further?

Reply:
shouldn't the examples all be no-phi?

Reply:
Ah, yeah. I just discussed that with @znatty22 and decided to add both no PHI and fake PHI examples. Thanks!

Reply:
i think it would be useful to have some examples with phi and test that they fail with the no-phi profiles. these examples will have phi but claim to be valid according to the no-phi profiles. they're invalid examples to make sure they don't pass validation.

Reply:
@liberaliscomputing ^ that's a good point Shahim makes. We can talk about how to do this. You'd have to change what's in `tests/data/profiles/valid/` and `tests/data/examples/invalid/`. Let's discuss before you do this! (I might change how tests work so this sort of thing is easier in future)